chatbot/vectordb.py

```diff
@@ -0,0 +1,318 @@
+## Import Python Packages
```

Comment:
What's this script for? Do we push data to the vector db here?

Reply:
In fact, I think it is an important file, which defines the chatbot_agent. In this file, we define/configure the GPT agent, do the prompt engineering, and input/output the data using vector db (an encapsulation of all the API, functions, prompts, etc.). And, I hope whenever we want to call the GPT-3.5, we should use the attribution of chatbot_agent.

Reply:
If so why we call this file as vectordb? Let's sync up offline to learn more.